sdk/security/keyvault/azkeys/client_test.go

```diff
@@ -537,69 +537,63 @@ func TestRecoverDeletedKey(t *testing.T) {
 }
 
 func TestReleaseKey(t *testing.T) {
-	for _, mhsm := range []bool{false, true} {
-		name := "KV"
-		if mhsm {
-			name = "MHSM"
-		}
-		t.Run(name, func(t *testing.T) {
-			client := startTest(t, mhsm)
-			key := createRandomName(t, "testreleasekey")
-
-			// retry creating the key because Key Vault sometimes can't reach the fake
-			// attestation service we use in CI for several minutes after deployment
-			var createResp azkeys.CreateKeyResponse
-			var err error
-			for i := 0; i < 5; i++ {
-				params := azkeys.CreateKeyParameters{
-					Curve: to.Ptr(azkeys.CurveNameP256K),
-					KeyAttributes: &azkeys.KeyAttributes{
-						Exportable: to.Ptr(true),
-					},
-					Kty: to.Ptr(azkeys.KeyTypeECHSM),
-					ReleasePolicy: &azkeys.KeyReleasePolicy{
-						EncodedPolicy: getMarshalledReleasePolicy(attestationURL),
-						Immutable:     to.Ptr(true),
-					},
-				}
-				createResp, err = client.CreateKey(context.Background(), key, params, nil)
-				if err == nil {
-					break
-				}
-				if i < 4 {
-					recording.Sleep(30 * time.Second)
-				}
+	t.Run("KV", func(t *testing.T) {
```

Comment:
So you're not going to test MHSM in the future? Why not just comment out the condition for now?

Reply:
Changed it so MHSM are just commented out for now.

Reply:
fixed in [d6507a9](https://github.com/Azure/azure-sdk-for-go/pull/21910/commits/d6507a98f031e9f4bcfcbb9a86b87b108186cbae)